test/testcreateprocwndstatew32.c

```diff
@@ -0,0 +1,180 @@
+/*
+  Copyright (C) 1997-2024 Sam Lantinga <slouken@libsdl.org>
+
+  This software is provided 'as-is', without any express or implied
+  warranty.  In no event will the authors be held liable for any damages
+  arising from the use of this software.
+
+  Permission is granted to anyone to use this software for any purpose,
+  including commercial applications, and to alter it and redistribute it
+  freely.
+*/
+/* Sample program:  Test SDL's ability to defuse the window state (minimized/maximized) that is applied on the initial window created. */
```

Comment:
Please note what bug this is a test case for.

Reply:
I have updated the description, is this more along the lines of what you were expecting here?